documentation/prometheus-mixin/alerts.libsonnet

```diff
@@ -225,6 +225,27 @@
               description: 'Prometheus %(prometheusName)s remote write is {{ printf "%%.1f" $value }}s behind for queue {{$labels.queue}}.' % $._config,
             },
           },
+          {
+            alert: 'PrometheusRemoteWriteDesiredShards',
+            expr: |||
+              # Without max_over_time, failed scrapes could create false negatives, see
+              # https://www.robustperception.io/alerting-on-gauges-in-prometheus-2-0 for details.
+              (
+                max_over_time(prometheus_remote_storage_shards_desired{%(prometheusSelector)s}[5m])
+              > on(job, instance) group_right
+                prometheus_remote_storage_shards_max{%(prometheusSelector)s})
+              )
+              == 1
```

Comment:
`== 1` requires a bool operator no? I don't think a simple `>` would work here.

Reply:
Only for scalar/scalar comparisons. > should work in here. the LHS however also needs a max_over_time.

Reply:
:+1: added